src/components/pages/StudentHome/StudentDashboard.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import '../../../styles/StudentStyles/index.less';
+
+const initialStudent = null;
+
+function StudentDashboard(props) {
+  const [student, setStudent] = useState(initialStudent);
```

Comment:
As this isn't being used yet, we're introducing a new linter warning with this line - let's comment it out and leave an additional comment noting its future necessity.

Reply:
copy that, it's been done!